_ml-commons-plugin/api.md

```diff
@@ -115,6 +115,120 @@ The API returns information on the model, the algorithm used, and the content fo
 }
 ```
 
+## Upload load a model
+
+Use the upload operation to upload a pre-existing or custom model and into a model index. ML commons splits the model into smaller chunks and saves those chunks into the model's index.
```

Comment:
Should this be "preexisting custom model", or can a custom model not be preexisting?

Reply:
Remove "preexisting".

Reply:
I don't understand what "preexisting" means in this context either. This requires some clarification. Isn't it enough to say the "upload command allows you to import and store your ML model on your OpenSearch server."